lib/rubocop/cop/style/one_line_conditional.rb

```diff
@@ -8,12 +8,17 @@ module Style
       class OneLineConditional < Cop
         include OnNormalIfUnless
 
-        MSG = 'Favor the ternary operator (?:) ' \
-              'over if/then/else/end constructs.'
+        MSG1 = 'Favor the ternary operator (?:) ' \
+               'over if/then/else/end constructs.'
```

Comment:
You can just replace `if` with `%s` and use `format(MSG, node.loc.keyword.source)` as argument to `add_offense`.